### PR TITLE
Added pt-br (Brazilian Portuguese) for "things-i-dont-know-as-of-2018"

### PR DESCRIPTION
**Pull request by lucasconstantino**
We did an effort at @TallerWebSolutions to translate "Things I Don’t Know as of 2018" post. We found it has many meaning in our company culture, and we thank you for this very motivational and encouraging content!

Ps.: most of the translation work was put-up by a junior developer of ours @ericvvf, so the commit goes in his name ;)

### CODE REVIEW

**Comment by lucasconstantino** [comment]
By the way: any chance we can re-publish the pt-br content in our company blog, provided we mention the authoring on the first lines?

**Comment by gaearon** [comment]
Sure, please link back to the original though!

File: src/pages/things-i-dont-know-as-of-2018.pt-br.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+title: Things I Don’t Know as of 2018
```
Comment (gaearon) [inline src/pages/things-i-dont-know-as-of-2018.pt-br.md]: Might want to translate the title?

Comment (lucasconstantino) [reply src/pages/things-i-dont-know-as-of-2018.pt-br.md]: There you go!

**Comment by lucasconstantino** [comment]
Thanks, I make sure to send here the link of the post once we re-publish it ;)

We'll probably still work on some translation reviews here, so perhaps you can keep this pull-request open for a couple of days ;)

**Comment by renatorib** [comment]
That's awesome!

**Pull request merged**